sdk/eventgrid/eventgrid-namespaces/samples/v1/javascript/namespaceActivities.js

```diff
@@ -1,17 +1,6 @@
-// Copyright (c) Microsoft Corporation.
```

Comment:
why was the license header removed?

Reply:
for testing?

Reply:
not sure, this was the result of running the `build:samples` task, I didn't touch the output at all

Reply:
May be a dev-tool issue. I will look into it.